Userland/Libraries/LibGfx/ImageFormats/WebPWriter.cpp

```diff
@@ -116,7 +116,10 @@ static ErrorOr<void> write_VP8L_image_data(Stream& stream, Bitmap const& bitmap)
     if (alphabet_sizes[0] > 288)
         return Error::from_string_literal("Invalid alphabet size");
 
-    for (int i = 0; i < 4; ++i) {
+    bool all_pixels_are_opaque = are_all_pixels_opaque(bitmap);
```

Comment:
Can't we pass the value from `encode`? Iterating over the whole image isn't cheap.